### PR TITLE
Revisions evolution boolean and enumeration ParameterTypes : Fix #272 #271

### LINKED ISSUE #272: [Subscription Dashboard] Revisions Evolution: Support of BooleanParameterType
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-WEB
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

The evolution graph should support to display revision for a BooleanParameterType

### PR DESCRIPTION
**Pull request by nabilatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

Fix #272 #271 

The evolution graph supports : 
- BooleanParameterType
image
- EnumerationParamerType
    enumertion-evolution
- EnumerationParamerType  that allows multiple selection :  
enumeration-multiple-evolution

### CODE REVIEW

File: COMETwebapp/Model/RevisionHistory.cs [hunk 1]
```diff
@@ -58,6 +65,6 @@ public RevisionHistory(int revisionNumber, ValueArray<string> valueArray)
         /// <summary>
         /// Actual value of the ParameterSubcriptionValueSet
         /// </summary>
-        public double? ActualValue { get; set; }
+        public Object ActualValue { get; set; }
```
Comment (antoineatrhea) [inline COMETwebapp/Model/RevisionHistory.cs:68]: please use the "object" instead of Object

**Review by antoineatrhea — CHANGES_REQUESTED** [review]
small change

**Comment by antoineatrhea** [comment]
@nabilatrhea Can you also provide screenshot of the 2 implemented solution?

**Comment by antoineatrhea** [comment]
For booleanParameterType, could it possible to have the y axis inversed? Like having the "-" at the bottom and the true on the top ?

**Review by jaimeatrhea — APPROVED** [review]

**Pull request merged**